tests/test-jetpack.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+// phpcs:disable PEAR.NamingConventions.ValidClassName.Invalid
+
+class VIP_Go__Core__Default_VIP_Jetpack_Version extends WP_UnitTestCase {
+	public function test__vip_default_jetpack_version() {
+		$versions_map = [
+			// WordPress version => Jetpack version
+			'5.8.6' => '10.9',
+			'5.9.0' => '11.4',
```

Comment:
WordPress doesn't exactly follow semver, so the versions would be like 5.9, 5.9.1 ,etc
```suggestion
			'5.9' => '11.4',
```